howto/vscode-debugging-daprd/README.md

```diff
@@ -0,0 +1,150 @@
+# Configuring Visual Studio Code for debugging with daprd
+
+While developing locally in VSCode, you might find it useful to be able to attach the debugger to your code that is utilizing Dapr. To do this, you can run the daprd process separately and then launch your code and attach the debugger. While this is a perfectly acceptable solution, it does require a few extra steps and some instruction to developers who might want to clone your repo and hit the "play" button to begin debugging.
+
+Using the ***tasks.json*** and ***launch.json*** files in Visual Studio Code, we can simplify the process and request that VSCode kick off the daprd process prior to launching the debugger.
+
+Let's get started!
+
+## Modifying launch.json configurations to include a preLaunchTask
+
+In your ***launch.json*** file add a **preLaunchTask** for each configuration that you want daprd launched. The **preLaunchTask** will reference tasks that you define in your tasks.json file. Here is an example for both Node and .NET Core. Notice the **preLaunchTask**s referenced: daprd-web and daprd-leaderboard.
```

Comment:
It would be better start with an example to works with the Dapr samples, rather than some random application names.  Here is what I created for the 1.Hello world example. That way people can just cut and paste this to get going. You can still keep this dotnet example for daprd-web, but just make it clear this is just showing how to do a .NET Core example along with a node example.

launch.json
{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "type": "node",
            "request": "launch",
            "name": "MyNodeApp",
            "preLaunchTask": "daprd-node",
            "skipFiles": [
                "<node_internals>/**"
            ],
            "program": "${workspaceFolder}\\1.hello-world\\app.js"
        }
    ]
}

tasks.json
{
    "version": "2.0.0",
    "tasks": [
        {
            "label": "daprd-node",
            "command": "daprd",
            "args": [
                "-dapr-id","mynode",
                "-app-port","3000",
                "-dapr-http-port","3500",
                "-dapr-grpc-port","52000",
                "-placement-address","localhost:50005"
            ],
            "isBackground": true,
            "problemMatcher": {
                "pattern": [
                    {
                      "regexp": ".",
                      "file": 1,
                      "location": 2,
                      "message": 3
                    }
                ],
                "background": {
                    "beginsPattern": "^.*starting Dapr Runtime.*",
                    "endsPattern": "^.*waiting on port.*"
                }
            }
        }
    ]
}

Reply:
I'm leaving the example as is for now since the Hello World application doesn't contain a dotnetcore service. I'll look into creating a simple example that we can link to later that utilizes dotnetcore, node and python and configures VS Code to debug all of them.